gradio/dev.py

```diff
@@ -0,0 +1,35 @@
+# Contains the function that runs when `gradio` is called from the command line. Specifically, allows
+# $ gradio app.py to run app.py in development mode where any changes reload the script.
+
+import os
+import sys
+
+from gradio import networking
+
+
+def main():
+    args = sys.argv[1:]
+    if len(args) == 0:
+        raise ValueError("No file specified.")
+    if len(args) == 1:
+        demo_name = "demo"
+    else:
+        demo_name = args[1]
+
+    path = args[0]
+    path = os.path.normpath(path)
+    if not (path == os.path.basename(path)):
+        raise ValueError("Must provide file name in the current directory")
```

Comment:
I can make it work like this in the repo folder. But cannot make it work in different folders.
```
 uvicorn dev_mode:app --reload --reload-dir "gradio"
←[32mINFO←[0m:     Will watch for changes in these directories: ['F:\\SecondaryDownloads\\git_repos\\gradio\\gradio']
←[32mINFO←[0m:     Uvicorn running on ←[1mhttp://127.0.0.1:8000←[0m (Press CTRL+C to quit)
←[32mINFO←[0m:     Started reloader process [←[36m←[1m23928←[0m] using ←[36m←[1mstatreload←[0m
←[33mWARNING←[0m:  The --reload flag should not be used in production on Windows.
←[32mINFO←[0m:     Started server process [←[36m19512←[0m]
←[32mINFO←[0m:     Waiting for application startup.
←[32mINFO←[0m:     Application startup complete.
←[33mWARNING←[0m:  StatReload detected file change in 'gradio\components.py'. Reloading...
←[33mWARNING←[0m:  The --reload flag should not be used in production on Windows.
←[32mINFO←[0m:     Started server process [←[36m7704←[0m]
←[32mINFO←[0m:     Waiting for application startup.
←[32mINFO←[0m:     Application startup complete.
←[33mWARNING←[0m:  StatReload detected file change in 'gradio\components.py'. Reloading...
```

Reply:
Gonna update